src/test/resources/net/sf/jabref/logic/exporter/ModsExportFormatTestBook.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<modsCollection xmlns:mods="http://www.loc.gov/mods/v3" xmlns:ns3="http://www.w3.org/2001/XMLSchema-instance" xmlns:ns4="http://www.w3.org/1999/xlink" ns3:schemaLocation="http://www.loc.gov/standards/mods/v3/mods-3-6.xsd">
+    <mods:mods>
+        <mods:genre>book</mods:genre>
+        <mods:identifier type="issn">123456789</mods:identifier>
+        <mods:subject>
+            <topic>note; and another note;</topic>
```

Comment:
I think the keywords should be separated and a new topic node should be created for each one instead of writing them in one node. Same as for the ModsExportFormatTestAllFields.xml.